src/common/transformations/src/transformations/common_optimizations/nop_elimination.cpp

```diff
@@ -837,9 +837,38 @@ ov::pass::NopSliceBeforeGatherElements::NopSliceBeforeGatherElements() {
     auto m = std::make_shared<pattern::Matcher>(gather, matcher_name);
     register_matcher(m, matcher_pass_callback);
 }
+ov::pass::EliminateSlice::EliminateSlice() {
+    MATCHER_SCOPE(EliminateSlice);
 
-ov::pass::NopStridedSlice::NopStridedSlice() {
-    MATCHER_SCOPE(NopStridedSlice);
+    auto input = pattern::any_input();
+    auto begin_const = pattern::wrap_type<ov::op::v0::Constant>();
+    auto end_const = pattern::wrap_type<ov::op::v0::Constant>();
+    auto step_const = pattern::wrap_type<ov::op::v0::Constant>();
+    auto axes = pattern::any_input();
```

Comment:
My understanding is that axes is an optional input and could be missing meaning all the axes are involved in slicing.